go.mod

```diff
@@ -4,10 +4,10 @@ go 1.15
 
 require (
 	github.com/Masterminds/semver/v3 v3.1.1
-	github.com/gin-gonic/gin v1.7.1
-	github.com/go-vela/pkg-executor v0.7.5-0.20210521154632-d53be352df36
+	github.com/gin-gonic/gin v1.7.2
+	github.com/go-vela/pkg-executor v0.7.5
 	github.com/go-vela/pkg-queue v0.7.5-0.20210402170103-bcd9ababfe88
-	github.com/go-vela/pkg-runtime v0.7.5-0.20210428154035-f007d6e59a10
+	github.com/go-vela/pkg-runtime v0.7.5
```

Comment:
`v0.7.5` was a backport fix:

https://github.com/go-vela/pkg-runtime/pull/115

Merging this would be a downgrade in functionality so we should block this.